flopy/discretization/grid.py

```diff
@@ -50,6 +50,8 @@ class Grid(object):
         in the spatial reference coordinate system
     rotation : float
         rotation angle of model grid, as it is rotated around the origin point
+    length_multiplier : float

```

Comment:
Should length_multiplier be removed from docstrings?

Reply:
Yes it should! Thanks for catching that.